commons-db/src/main/java/com/palantir/nexus/db/DBType.java

```diff
@@ -42,17 +41,12 @@ public boolean supportsGIS() {
         return hasGIS;
     }
 
-    private DBType(String defaultUrl, String driver, String testQuery, boolean hasGIS) {
-        this.defaultUrl = defaultUrl;
+    private DBType(String driver, String testQuery, boolean hasGIS) {
         this.driver = driver;
         this.testQuery = testQuery;
         this.hasGIS = hasGIS;
     }
 
-    public String getDefaultUrl() {
```

Comment:
Do we need to break this? I am not entirely aware of the usecase here, but looks like we should use the configured URL if its specified or default to the default url,

Reply:
This is legacy code that we used to format the URLs. The logic now lives in OracleConnectionConfig.getUrl() etc., so getDefaultUrl() is not used by Atlas. It is still used in our internal product, but I have a change which makes it go through the new code path instead.